problems/5.loops.js

```diff
@@ -2,17 +2,29 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
-
+function evenNumbers(num) {
+  let arr = [];
+  for(let i = 0; i <= num; i++){
```

Comment:
How would changing the step to i += 2 improve things?

Reply:
I feel like now looking back at the assessment, i would redo most of my answers in a much more cleaner and shorter way.. i += 2 is the shorter and faster way.. no need for the if statement.